00-members.tf

```diff
@@ -184,6 +184,7 @@ locals {
     local.vision_opencv_team,
     local.vision_team,
     local.visp_team,
+    local.visualization_team,
```

Comment:
For what it is worth, I'm not a huge fan of calling this team `visualization_team`.  There are a lot of repositories in https://github.com/ros-visualization, and the members of this team aren't necessarily members of all of the teams.

Reply:
@clalancette any suggestion of another name would have been acceptable to me. In the absence of one, I'm going to merge as is but these team names don't propagate out of here so it's perfectly fine to change them later.

Reply:
OK, that's fair.  I can open a PR to rename this.